core-v1/src/main/scala/algebra/WindowComparable.scala

```diff
@@ -0,0 +1,60 @@
+package com.rallyhealth.vapors.v1
+
+package algebra
+
+import data.{Justified, Window}
+import debug.HasShow
+
+import cats.data.NonEmptyList
+import cats.{Id, Show}
+
+trait WindowComparable[F[_], OP[_]] {
+
+  // TODO: Should this take OP[V] and OP[Window[V]]? What about the F[_] wrapper?
+  //       For now, I am only taking the OP[V] because if the goal is to serialize the value and the window,
+  //       then that is the missing piece.
+  def withinWindow[V](
+    value: F[V],
+    window: F[Window[V]],
+  )(implicit
+    opV: OP[V],
+  ): F[Boolean]
```

Comment:
I'm not really using the `OP[V]` here, but the idea is that if the `OP` provides an encoder for `V`, then I can serialize the window and the value as json (or whatever type of encoder it is). I'm not really using it right now though. I'm gonna leave it until it becomes an impediment.